src/deadline/cinema4d_adaptor/Cinema4DClient/cinema4d_handler.py

```diff
@@ -50,6 +50,15 @@ def __init__(self, map_path: Callable[[str], str]) -> None:
         self.take = "Main"
         self.map_path = map_path
 
+    def _remap_assets(self):
+        asset_list = []
+        c4d.documents.GetAllAssetsNew(self.doc, allowDialogs=False, lastPath="", assetList=asset_list)
+        for asset in asset_list:
+            asset_owner = asset.get("owner")
+            asset_param_id = asset.get("paramId")
+            if asset_owner and asset_param_id:
+                asset_owner[asset.get("paramId")] = self.map_path(asset.get("filename"))
```

Comment:
Nit: We could re-use `asset_param_id` instead of calling `asset.get("paramId")`?

Reply:
Good catch, I'll update it